src/load/azext_load/data_plane/utils/validators.py

```diff
@@ -402,3 +402,65 @@ def validate_disable_public_ip(namespace):
         namespace.disable_public_ip = True
     else:
         namespace.disable_public_ip = False
+
+
+def validate_autostop_enable_disable(namespace):
+    if namespace.autostop is None:
+        return
+    if not isinstance(namespace.autostop, str) or namespace.autostop.casefold() not in ["enable", "disable"]:
+        raise InvalidArgumentValueError(
+            f"Invalid autostop type: {type(namespace.autostop)}. Allowed values: enable, disable"
+        )
+    if namespace.autostop.casefold() not in ["disable"]:
+        namespace.autostop = True
+    else:
+        namespace.autostop = False
+
+
+def validate_autostop_error_rate_time_window(namespace):
+    if namespace.autostop_error_rate_time_window is None:
+        return
+    if not isinstance(namespace.autostop_error_rate_time_window, int):
+        raise InvalidArgumentValueError(
+            f"Invalid autostop-error-rate-time-window type: {type(namespace.autostop_error_rate_time_window)}"
+        )
+    if namespace.autostop_error_rate_time_window < 0:
+        raise InvalidArgumentValueError(
+            "Autostop error rate time window should be greater than or equal to 0"
+        )
+
+
+def validate_autostop_error_rate(namespace):
+    if namespace.autostop_error_rate is None:
+        return
+    if not isinstance(namespace.autostop_error_rate, float):
+        raise InvalidArgumentValueError(
```

Comment:
I think we should also check for int here. 
Since "100" is giving false for `isinstance(100, float)`

Reply:
I have specified the argument type as float in argtypes.py so CLI automatically is able to convert 100 to 100.0 and this validation passes

Reply:
moreover, I'm not sure unit tests are possible for this because the cmd itself fails when type check fails on the argument and catchable exception is thrown

Reply:
since error case isn't possible on this, added the happy case where integer error rate works from cli

Reply:
Got it okay, thanks!